Release/1.2.0

adatzer opened the pull request.

paulboocock commented on template.tpl:
```diff
@@ -381,7 +381,7 @@ ___TEMPLATE_PARAMETERS___
         ],
         "displayName": "Cookie Name",
-        "defaultValue": "sp",
+        "defaultValue": "_sp_",
```
This is a humdinger. Will changing this now, for all those already using `sp` mean that all their cookies get regenerated? Or does the template tell you about this when the upgrade happens in the UI? :-/

adatzer replied:
I was hoping this is pointed by the UI upon available update, but i haven't checked for this specific change. The UI offers a chance to review the changes, which may not be enough as a pointer. So you are right, i don't know if we can be 100% sure. Do you think we should drop this change until a v2 for example?

paulboocock replied:
instead of changing this, how about we update the help text to identify this is a different default to the trackers defaults, if you want it to be the same set it to `_sp_`?

adatzer commented:
Thanks again for reviewing @paulboocock !
Changes:

- reverted the change for default cookieName prefix. Instead updated the help text as you mentioned.
- cherry-picked and renamed the commit from #12  , and also added the same help text as you mentioned in [the corresponding Tag PR](https://github.com/snowplow/snowplow-gtm-tag-template-v3/pull/11).
- Added Prepare-for-release commit

The plan is to release first [the Tag v1.1.0](https://github.com/snowplow/snowplow-gtm-tag-template-v3/pull/10) and then the settings Variable here.
